calcite-qml/demo/main.cpp

```diff
@@ -0,0 +1,74 @@
+/*******************************************************************************
+ *  Copyright 2012-2021 Esri
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ ******************************************************************************/
+#include <QDir>
+#include <QGuiApplication>
+#include <QQmlApplicationEngine>
+#include <QQuickWindow>
+#include <QQuickStyle>
+
+//------------------------------------------------------------------------------
+
+int
+main(int argc, char* argv[])
+{
+  qDebug() << "Initializing application";
+
+  QGuiApplication::setAttribute(Qt::AA_EnableHighDpiScaling);
+  QGuiApplication app(argc, argv);
+
+  // Use of Esri location services, including basemaps and geocoding, requires
+  // either an ArcGIS identity or an API key. For more information see
+  // https://links.esri.com/arcgis-runtime-security-auth.
+
+  // 1. ArcGIS identity: An ArcGIS named user account that is a member of an
+  // organization in ArcGIS Online or ArcGIS Enterprise.
+
+  // 2. API key: A permanent key that gives your application access to Esri
+  // location services. Create a new API key or access existing API keys from
+  // your ArcGIS for Developers dashboard
+  // (https://links.esri.com/arcgis-api-keys).
+
+  const QString apiKey =
+    QStringLiteral("");
```

Comment:
🔧  I believe our templates have been updated to not use QStringLiteral anymore as it was raising a warning in Creator

```suggestion
  const QString apiKey =
    QString("");
```